docs/100.yaml

```diff
@@ -343,6 +343,8 @@
   title  : Pick’s Theorem
 93:
   title  : The Birthday Problem
+  decl   : fintype.card_inj
```

Comment:
```suggestion
  decl   : birthday
```

Reply:
this wouldn't lint; all others that are in `archive` put it under `links`. It also kinda obscures the proof, as it's non-obvious that `norm_num` calls it; but not sure

Reply:
(yes there is a linter for this, as I found out from many CI failures...)

Reply:
I think you should put it under links then; possibly in addition to what you have right now.